documentation/Forms/FormViewer.md

```diff
@@ -0,0 +1,577 @@
+* "Protected A"
+* "Protected B"
+* "Unclassified"
+
+## <a name="internalTitleEn"></a>2. [Optional] Property `GC Forms Schema - Alpha > internalTitleEn`
+
+| Type | `string` |
+| ---- | -------- |
+
+**Description:** The internal title of the form in English
+
+## <a name="internalTitleFr"></a>3. [Optional] Property `GC Forms Schema - Alpha > internalTitleFr`
+
+| Type | `string` |
+| ---- | -------- |
+
+**Description:** The internal title of the form in French
+
+## <a name="publishingStatus"></a>4. [Required] Property `GC Forms Schema - Alpha > publishingStatus`
+
+| Type | `boolean` |
+| ---- | --------- |
+
+**Description:** If set to false, the link to the form will not appear on the main page, rather under 'sandbox'
+
+## <a name="form"></a>5. [Required] Property `GC Forms Schema - Alpha > form`
+
+| Type                      | `object`                                                                  |
+| ------------------------- | ------------------------------------------------------------------------- |
+| **Additional properties** | [[Any type: allowed]](# "Additional Properties of any type are allowed.") |
+
+| Property                                  | Pattern | Type            | Deprecated | Definition | Title/Description                                                        |
+| ----------------------------------------- | ------- | --------------- | ---------- | ---------- | ------------------------------------------------------------------------ |
+| + [layout](#form_layout )                 | No      | array of number | No         | -          | A list of element IDs that determines the order or elements in the form. |
+| + [elements](#form_elements )             | No      | array           | No         | -          | A list of elements contained within the form.                            |
+| + [titleEn](#form_titleEn )               | No      | string          | No         | -          | The displayed title of the form to the user in English                   |
+| + [titleFr](#form_titleFr )               | No      | string          | No         | -          | The displayed title of the form to the user in French                    |
+| - [brand](#form_brand )                   | No      | object          | No         | -          | -                                                                        |
+| - [emailSubjectEn](#form_emailSubjectEn ) | No      | string          | No         | -          | The email subject used in the Notify template in English.                |
+| - [emailSubjectFr](#form_emailSubjectFr ) | No      | string          | No         | -          | The email subject used in the Notify template in French.                 |
+| - [endPage](#form_endPage )               | No      | object          | No         | -          | Content that will be displayed after a form is submitted.                |
+
+### <a name="form_layout"></a>5.1. [Required] Property `GC Forms Schema - Alpha > form > layout`
```

Comment:
I don't believe the layout is a required property but perhaps I'm mistaken

Reply:
Not certain if it's required or not either, but in the FormProperties interface, it's not optional.

https://github.com/cds-snc/platform-forms-client/blob/6ec78bba65ece67432236017f571aa8531d3dcb4/lib/types/form-types.ts#L106

Reply:
It's a required property.  The `layout` field is what FormBuilder uses to determine what `elements` are used and in what order they're placed.